backend/src/common/enums/experience.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum Experience {
+  INTERESTED,
```

Comment:
Do we need to set the string value here? (I'm sure this is right but this part of typescript confuses me still)

Reply:
(I just wasn't sure if they would be initialized w/o the  = "enum string value")

Reply:
They would be, this is equivalent to
```
enum Experience {
INTERESTED = 'INTERESTED'
}
```

The thought is that we have an enum for what they should be stored in the database as (all caps, no spaces, underscores) and a different enum if we need the user friendly name for things. That latter enum for each thing might be frontend only, idk yet.

Reply:
cool makes sense ty!